buidler.config.js

```diff
@@ -18,11 +18,5 @@ module.exports = {
       enabled: true,
       runs: 200
     }
-  },
-  networks: {
-    soliditycoverage: {
-      gas: 9000000,
-      url: 'http://localhost:8555'
-    }
   }
```

Comment:
The plugin will set this and it would overwrite this gas value to something much larger because the instrumented contracts are quite a bit larger and the number of opcodes run per tx is greater too.